sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_client_async.py

```diff
@@ -3,10 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 #--------------------------------------------------------------------------
-
-# TODO: check this
-# pylint: disable=super-init-not-called,too-many-lines
-
+# TODO: Check types of kwargs (issue exists for this)
```

Comment:
maybe add the link to the issue?

Reply:
It is linked in the documentation of this PR, let me try to get it to show up